inginious/frontend/i18n/es/LC_MESSAGES/messages.po

```diff
@@ -1672,6 +1672,167 @@ msgstr "Enviar"
 msgid "Grader"
 msgstr "Calificador"
 
+#: inginious/frontend/plugins/grader_generator/pages/grader_form.py:25
+msgid "'Maximum diff lines' must be an integer"
+msgstr "'Máximo líneas de diferencia' debe ser entero"
```

Comment:
That sounds weird. Maybe `Líneas de diferencia máxima` or `Número máximo de líneas de diferencia` or `El Máximo de líneas de diferencia`